e2e/various-suite/return-to-previous.spec.ts

```diff
@@ -0,0 +1,91 @@
+import { e2e } from '../utils';
+
+describe('ReturnToPrevious button', () => {
+  before(() => {
+    e2e.flows.login(Cypress.env('USERNAME'), Cypress.env('PASSWORD'));
+    cy.window().then((win) => {
+      win.localStorage.setItem('grafana.featureToggles', 'returnToPrevious=1');
+    });
+
+    // Create a new alert rule with linked dashboard
+    cy.visit('/alerting/list?search=');
+
+    cy.get('[data-testid="data-testid Call to action button New alert rule"]').click();
+    cy.get('[id="name"]').type('test rule');
+
+    cy.get('[data-testid="data-testid create folder"]').click();
+    cy.get('[id="folderName"]').last().type('test folder {enter}');
+
+    cy.get('[data-testid="data-testid create evaluation group"]').click();
+    cy.get('[id="group"]').last().type('test group');
+    cy.get('[id="eval-every-input"]').type('5m');
+    cy.get('[type="submit"]').click();
+
+    cy.get('[data-testid="data-testid save rule exit"]').click();
+    cy.wait(600); // TODO: use await instead?
```

Comment:
we should never do this. can we wait for something to appear indicating success instead? 🤔